app/views/usergroups/_form.html.erb

```diff
@@ -54,7 +54,7 @@
         </div>
         <div class="children_fields">
           <%= new_child_fields_template(f, :external_usergroups, { :partial => 'external' } ) %>
-          <%= add_child_link '+ ' + _('Add external user group'), :external_usergroups, { :class => "info disable-unsupported", :title => _('link external user group with this user group') } %>
+          <%= add_child_link '+ ' + _('Add external user group'), :external_usergroups, { :class => "info", :title => _('link external user group with this user group') } %>
```

Comment:
The class obviously got here just by copy-pasting. We used `disable-unsupported` only in hosts but it's no longer necessary. I replaced it with variable `new_host?` in the partials.